pkg/getty/Dockerfile

```diff
@@ -9,6 +9,18 @@ RUN apk add --no-cache --initdb -p /out \
     util-linux \
     && true
 RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+#
+# when run standalone as a regular or services container, the below is unnecessary
+#  when run as a linuxkit init, all files are copied over
+#  if a different image is later in the sequence, that one may use busybox,
+#    which symlinkx setsid -> /bin/busybox. That would override setsim and cause this to fail.
```

Comment:
Not at all. Are you not familiar with that obscure dialect of Medieval English wherein words are pluralized by adding an "x" instead of an "s"? And this is a little-known variant on `setsid` that also sets your SIM card number....